packages/rx_bloc_cli/mason_templates/bricks/feature_deeplink/__brick__/lib/l10n/network_lookup/deep_link_lookup.dart

```diff
@@ -21,8 +21,6 @@ class AppI18nDeepLinkLookup extends I18nFeatureDeepLinkLookup {
           bundledI18n = I18nFeatureDeepLink(I18nFeatureDeepLinkLookup_bg());
           break;
         case 'en':
-          bundledI18n = I18nFeatureDeepLink(I18nFeatureDeepLinkLookup_en());
```

Comment:
Please make sure such changes are addressed in separate PRs. From what I see this is not related to QR code integration, but feel free to correct me if I'm wrong